### PR TITLE
Target create

### PR DESCRIPTION
**Pull request by leherreras**
As a user I should be able to create a new target so I can find people with similar interests in a specific area

### CODE REVIEW

File: spec/models/target_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  it 'validates name presence' do
+    expect(target.title).to be_present
+    expect(target.radius).to be_present
+    expect(target.latitude).to be_present
+    expect(target.longitude).to be_present
+  end
+
+  describe 'associations' do
+    it { should belong_to(:user) }
```
Comment (enzofab91) [inline spec/models/target_spec.rb]: I'm pretty sure that `should` is deprecated.

```suggestion
it { is_expected.to belong_to(:user) }
```

https://github.com/rspec/rspec-expectations/blob/master/Should.md

**Review by enzofab91 — APPROVED** [review]
There are some code not related to this feature. Take care in next PRs

File: app/views/api/v1/targets/_target.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json.extract! target, :id, :title, :radius, :latitude, :longitude, :topic_id, :user_id, :created_at, :updated_at
+json.url api_v1_target_url(target, format: :json)
```
Comment (sebastiancaraballo) [inline app/views/api/v1/targets/_target.json.jbuilder:4]: What is this `url` attribute for?

**Review by sebastiancaraballo — APPROVED** [review]

File: app/views/api/v1/targets/_target.json.jbuilder (hunk 2 above)
Comment (leherreras) [reply app/views/api/v1/targets/_target.json.jbuilder:4]: this code was create automatically for scaffold command

**Pull request merged**